moebot_bot/util/util.go

```diff
@@ -132,9 +137,22 @@ func OpenPollMessage(poll *db.Poll, user *discordgo.User) string {
 func ClosePollMessage(poll *db.Poll, user *discordgo.User) string {
 	var message string
 	if poll.Open {
-		message = user.Mention() + " closed " + UserIdToMention(poll.UserUid) + "'s poll **" + poll.Title + "**!\n"
+		if user.ID == poll.UserUid {
+			message = user.Mention() + " closed his poll"
```

Comment:
Let's make sure to use they/their/them when referring to users, especially to be consistent with the rest of the user messaging

Reply:
Fixed with https://github.com/camd67/moebot/pull/12/commits/5c34c0918b30efab2ab10a52b211d62395136488